src/open_company_web/actions.cljs

```diff
@@ -77,12 +77,14 @@
 
 (defmethod dispatcher/action :auth-settings [db [_ body]]
   (if body
```

Comment:
yup, if we want to return special error messages we need to change this but we can still assoc the body here and then show an error message in the UI. For now it should work w/ positive and negative responses